tests/lib/nm/translator_test.py

```diff
@@ -110,7 +113,8 @@ def test_nm2api_bond_info():
 
 
 def test_iface_admin_state(NM_mock):
-    NM_mock.DeviceState.ACTIVATED = 'ACTIVATED'
-    admin_state = nm.translator.Nm2Api.get_iface_admin_state('ACTIVATED')
+    NM_mock.DeviceState.ACTIVATED = 100
+    NM_mock.DeviceState.IP_CONFIG = 70
+    admin_state = nm.translator.Nm2Api.get_iface_admin_state(80)
```

Comment:
Can you name this value 80 like you did in the other test?

Reply:
Done.